assets/src/edit-story/components/inspector/document/slug.js

```diff
@@ -88,7 +91,12 @@ function SlugPanel() {
         />
       </Row>
       <HelperText>
-        <Permalink rel="noopener noreferrer" target="_blank" href={link}>
+        <Permalink
+          rel="noopener noreferrer"
+          target="_blank"
+          href={link}
+          data-testid="link"
```

Comment:
Please remove test-id here. I don't see this used in a test below actually, but if you need it, please use:

```js
getByRole('link', { name: new RegExp(permalinkConfig.prefix) });
```

(and of course make sure there's something rather unique-looking in `permalinkConfig.prefix` as there would be in the real-world editor)